saleor/warehouse/availability.py

```diff
@@ -71,14 +71,16 @@ def check_stock_quantity_bulk(
         if not stocks:
             insufficient_stocks.append(
                 InsufficientStockData(
-                    variant=variant, available_quantity=available_quantity
+                    variant=variant,
+                    available_quantity=available_quantity,  # type: ignore
                 )
             )
         elif variant.track_inventory:
-            if quantity > available_quantity:
+            if quantity > available_quantity:  # type: ignore
```

Comment:
Why are we adding here `# type: ignore`?

Reply:
Because I've got mypy errors:
`Argument "available_quantity" to "InsufficientStockData" has incompatible type "Union[Callable[[], Any], int]"; expected "Optional[int]"  [arg-type].`
I have no idea why. I didn't change any line of code within those files nor related files.